src/python/pants/core/goals/fix_integration_test.py

```diff
@@ -35,3 +35,48 @@ def run() -> None:
         # Re-run and confirm that the file was fixed.
         run()
         assert good_content == read_file(f)
+
+
+@ensure_daemon
```

Comment:
I think you can skip this probably. It's only important for the above because we are testing file watching with making an edit to it.

The benefit of removing is that we have one fewer integration test, so it's less of a CI slowdown.